shared/settings/account/email-phone-row.tsx

```diff
@@ -40,13 +42,24 @@ const _EmailPhoneRow = (props: Kb.PropsWithOverlay<Props>) => {
     return null
   }
 
+  // less than 30 minutes ago
+  const hasRecentVerifyEmail =
+    props.lastVerifyEmailDate && new Date().getTime() / 1000 - props.lastVerifyEmailDate < 30 * 60
+
   let subtitle = ''
-  if (props.type === 'email' && props.primary) {
-    subtitle = addSpacer(subtitle, 'Primary')
-    // TODO 'Check your inbox' if verification email was just sent
-  }
-  if (!props.searchable) {
-    subtitle = addSpacer(subtitle, 'Not searchable')
+
+  if (isMobile && hasRecentVerifyEmail) {
+    subtitle = 'Check your inbox'
+  } else {
+    if (props.type === 'email' && props.primary) {
+      subtitle = addSpacer(subtitle, 'Primary')
+    }
+    if (!props.searchable) {
+      subtitle = addSpacer(subtitle, 'Not searchable')
+    }
+    if (hasRecentVerifyEmail) {
+      subtitle = addSpacer(subtitle, 'Check your inbox')
```

Comment:
Maybe this should go first on desktop so "Check your inbox" has some context next to "UNVERIFIED"
<img width="349" alt="image" src="https://user-images.githubusercontent.com/11968340/62949548-5fc4b280-bdb4-11e9-9552-7f6dd2ed4215.png">